src/encoded/static/components/browse/components/StackedBlockTable.js

```diff
@@ -706,6 +706,15 @@ export class FileEntryBlock extends React.Component {
         return this.props.selectedFiles[accessionTriple];
     }
 
+    fileTypeSummary(file = this.props.file){
```

Comment:
Unrelated to current branch/edits. Better contents for 'File Info' row on stacked raw/processed files tables.